packages/server/src/__tests__/runQuery.test.ts

```diff
@@ -837,7 +837,7 @@ describe('request pipeline life-cycle hooks', () => {
               message: 'Syntax Error: Expected Name, found "}".',
               extensions: {
                 code: 'GRAPHQL_PARSE_FAILED',
-                http: { status: 400, headers: new HeaderMap() },
+                http: { status: 400, headers: expect.any(HeaderMap) },
```

Comment:
I actually don't understand why this worked before or why my change required this test update, but this is what we should've had here to begin with.